src/test/kotlin/com/exactpro/th2/check1/rule/AbstractCheckTaskTest.kt

```diff
@@ -40,9 +44,16 @@ abstract class AbstractCheckTaskTest {
         )
     }
 
-    fun constructMessage(sequence: Long = 0, alias: String = SESSION_ALIAS, type: String = MESSAGE_TYPE, direction: Direction = FIRST): Message.Builder = Message.newBuilder().apply {
+    fun constructMessage(
+        sequence: Long = 0,
+        alias: String = SESSION_ALIAS,
+        type: String = MESSAGE_TYPE,
+        direction: Direction = FIRST,
+        timestamp: Timestamp? = Timestamp.getDefaultInstance()
```

Comment:
Why do you need a nullable timestamp for the message? If you using default value for argument there is no need to use nullable type (in most cases)